pom.xml

```diff
@@ -1857,6 +1857,9 @@
                 <param>com.puppycrawl.tools.checkstyle.checks.NoCodeInFileCheck*</param>
                 <param>com.puppycrawl.tools.checkstyle.checks.OrderedPropertiesCheck*</param>
                 <param>com.puppycrawl.tools.checkstyle.checks.OuterTypeFilenameCheck*</param>
+                <param>
+                  com.puppycrawl.tools.checkstyle.checks.naming.PatternVariableNameCheck
```

Comment:
Why not `com.puppycrawl.tools.checkstyle.checks.naming.PatternVariableNameCheck*`?

Reply:
Fixed.